### ISSUE TITLE
[a11y] This feature is not accessible

### ISSUE THREAD

**Issue by zamentur** [marcosegato/grav-plugin-backtotop#7]
Screen reader are not able to read alternative text cause there isn't.

I have added aria-label and <text> on my side and it seems to work (at least with orca):
```
<a href="#top" id="return-to-top" aria-label="{{ "THEME_QUARK.FOOTER.TOP"|t|e }}">

{% spaceless %}
    <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" fill-rule="evenodd" viewBox="0 0 24 24" clip-rule="evenodd">
        <path d="M11 2.206l-6.235 7.528-.765-.645 7.521-9 7.479 9-.764.646-6.236-7.53v21.884h-1v-21.883z"/>
        <text class="sr-only">{{ "THEME_QUARK.FOOTER.TOP"|t|e }}</text>
    </svg>

{% endspaceless %}

</a>

```

**Comment by marcosegato** [comment]
Thanks so much @zamentur, patch has been released to 0.6.2
